R/Rules-validity.R

```diff
@@ -435,8 +431,12 @@ v_stopping_mtd_cv <- function(object) {
     "target must be probability value from (0, 1) interval"
   )
   v$check(
-    test_probability(object@thresh_cv / 100, bounds_closed = FALSE),
+    test_probability(object@thresh_cv / 100, bounds_closed = c(FALSE, TRUE)),
```

Comment:
OK I guess I confused with this here - maybe good if you double check that the tests for this are correct?

Reply:
I think this is ok, so `tresh_cv = 100` means that we stop for sure, regardless of the posterior dose samples, as:
```
do_stop <- (mtd_cv <= stopping@thresh_cv)
```
and `mtd_cv` takes on value from `[0, 1]`.